src/System.Private.ServiceModel/src/Internals/System/Runtime/TaskHelpers.cs

```diff
@@ -311,8 +337,8 @@ public bool IsCompleted
             get
             {
                 return (TaskScheduler.Current == TaskScheduler.Default) &&
-                       (SynchronizationContext.Current == null ||
-                       (SynchronizationContext.Current.GetType() == typeof(SynchronizationContext)));
+                    (SynchronizationContext.Current == null ||
+                    (SynchronizationContext.Current.GetType() == typeof(SynchronizationContext)));
```

Comment:
This is existing code, looks like it got reformatted. This works fine in Net Native as this is easily discoverable by the analysis engine. Also, I'm not needing runtime types so I don't think even reflection information is required.